src/uu/pinky/src/pinky.rs

```diff
@@ -120,6 +141,17 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     }
 }
 
+#[cfg(target_os = "openbsd")]
```

Comment:
please add a comment explaining why it is unspported

Reply:
OK, I will add a comment to explain why it's unsupported on OpenBSD.